wasmbinding/stargate_whitelist.go

```diff
@@ -36,6 +36,7 @@ func init() {
 
 	// bank
 	StargateWhitelist.Store("/cosmos.bank.v1beta1.Query/Balance", &banktypes.QueryBalanceResponse{})
+	StargateWhitelist.Store("/cosmos.bank.v1beta1.Query/AllBalances", &banktypes.QueryAllBalancesResponse{})
```

Comment:
I don't think we should put this in our whitelist or encourage people to use it.

Reply:
kk makes sense, moving this to test case set up